_posts/2020-05-27-ols1-wrapup.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: post
+title: End of OLS-1 and OLS-2 Announcement post
+authors: 
+- yochannah
+- bebatut
+- malvikasharan
+image: https://images.unsplash.com/photo-1577036421869-7c8d388d2123?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1950&q=80
+---
+
```

Comment:
```suggestion
{% assign people = site.data.people %}
{% assign projects = site.data.ols-1-projects %}

{% assign grad-participants = '' %}
{% assign grad-mentors = '' %}
{% assign grad-projects = '' %}
{% assign all-participants = '' %}
{% assign all-mentors = '' %}
{% for project in projects %}
    {% if project.graduated %}
        {% capture grad-projects %}{{ grad-projects}}, {{ project.name }}{% endcapture %}
        {% for p in project.participants %}
            {% capture grad-participants %}{{ grad-participants}}, [{{ people[p].first-name }} {{ people[p].last-name }}](/ols-1/projects-participants#{{ p }}){% endcapture %}
        {% endfor %}
        {% for m in project.mentors %}
            {% capture grad-mentors %}{{ grad-mentors }}, [{{ people[m].first-name }} {{ people[m].last-name }}](/ols-1#{{ m }}){% endcapture %}
        {% endfor %}
    {% endif %}
    {% for p in project.participants %}    
        {% capture all-participants %}{{ all-participants}}, [{{ people[p].first-name }} {{ people[p].last-name }}](/ols-1/projects-participants#{{ p }}){% endcapture %}
    {% endfor %}
    {% for m in project.mentors %}
        {% capture all-mentors %}{{ all-mentors }}, [{{ people[m].first-name }} {{ people[m].last-name }}](/ols-1#{{ m }}){% endcapture %}
    {% endfor %}
{% endfor %}

{% assign grad-projects = grad-projects | remove_first: ', ' | split: ", " | uniq | sort %}
{% assign grad-participants = grad-participants | remove_first: ', ' | split: ", " | uniq | sort %}
{% assign grad-mentors = grad-mentors | remove_first: ', ' | split: ", " | uniq | sort %}
{% assign all-participants = all-participants | remove_first: ', ' | split: ", " | uniq | sort %}
{% assign all-mentors = all-mentors | remove_first: ', ' | split: ", " | uniq | sort %}
```

Reply:
To collect lists of graduated projects, people, mentors and full lists too

Reply:
Thank you @bebatut. It would have taken me hours to add that.